### PR TITLE
Header and footer. Woo (exclamation point)

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1,2 +1,4 @@
 @import "~@edx/edx-bootstrap/sass/edx/theme";
 @import "~bootstrap/scss/bootstrap";
+
+@import "~@edx/frontend-component-footer/src/lib/scss/site-footer";
```
Comment (jlajoie) [inline src/App.scss]: nit: Why blank line

File: src/index.jsx [hunk 2]
```diff
@@ -30,6 +31,24 @@ const App = () => (
...
+          supportUrl={process.env.SUPPORT_URL}
+          contactUrl={process.env.CONTACT_URL}
+          openSourceUrl={process.env.OPEN_SOURCE_URL}
+          termsOfServiceUrl={process.env.TERMS_OF_SERVICE_URL}
+          privacyPolicyUrl={process.env.PRIVACY_POLICY_URL}
+          facebookUrl={process.env.FACEBOOK_URL}
+          twitterUrl={process.env.TWITTER_URL}
+          youTubeUrl={process.env.YOU_TUBE_URL}
+          linkedInUrl={process.env.LINKED_IN_URL}
+          googlePlusUrl={process.env.GOOGLE_PLUS_URL}
```
Comment (jlajoie) [inline src/index.jsx]: Should we leave a not that this is going away soon?

File: src/App.scss (hunk 1 above)
Comment (Dillon-Dumesnil) [reply src/App.scss]: Separates by app scss we are pulling in. If we end up using more apps, the lines will help

File: src/index.jsx (hunk 2 above)
Comment (Dillon-Dumesnil) [reply src/index.jsx]: :shrug: Nah. We'll just remove it when it's time to

**Pull request merged**